packages/apps/transfer/src/constants/dev-options.ts

```diff
@@ -0,0 +1,34 @@
+import { SystemIcon } from '@kadena/react-ui';
+
+import { DevOption } from './kadena';
+
+import React from 'react';
+
+export interface IDevOption {
+  title: string;
+  text: string;
+  icon: React.FC;
+  tag?: string;
+}
+
+export const devOptions: {
+  [Key in DevOption]: IDevOption;
+} = {
+  BASIC: {
+    title: 'Basic User Interface',
```

Comment:
Translations

Reply:
Since I can't use `useTranslations` hook outside functional components, maybe I will remove this constants file and integrate this code in the `packages/apps/transfer/src/components/Global/OptionsModal/OptionsModal.tsx`